app_context/app_context.go

```diff
@@ -116,6 +116,19 @@ func (self *baseAppContext) SetLogger(logger logger.CtxLogger) AppContext {
 	return self
 }
 
+func (self *baseAppContext) TiltEnvironment() (string, error) {
```

Comment:
(ie, this should be a part of NewAppContext() call... and error returned there if env is bogus)